coursedashboards/templates/handlebars/page-top.html

```diff
@@ -6,12 +6,12 @@
             <div class="container">
             <a href="#maincontent" id="shortcut" class="visually-hidden">skip to main content</a>
             <a href="http://www.uw.edu" class="uw-logo" style="background: transparent url('{% static "coursedashboards/img/uw-sprite-mini.png" %}') no-repeat;">University of Washington</a>
-            <div class="userinfo">
-                <span class="visually-hidden logout">Logged in as: </span><span><i class="ion-ios-person" aria-hidden="true"></i> {{netid}}</span> <a href="#">logout</a>
-            </div>
             <div class="row">
                 <h1 class="sitename"><span class="hide-sm">Course</span> Dashboard 
                 </h1>
+                <div class="userinfo">
+                    <span class="visually-hidden logout">Logged in as: </span><span><i class="ion-person" aria-hidden="true"></i> {{netid}}</span> | <i class="ion-log-out"></i> <a href="#" style="text-decoration: underline; color:#fff;">Sign out</a>
```

Comment:
You can refactor this later, but don't forget to pull out inline styles into stylesheets.